### ISSUE TITLE
Quotes for additional argument changed in V.077

### ISSUE THREAD

**Issue by guidotripaldi** [grych/drab#117]
Previously, you could freely choose how to use double and single quotes when passing an argument to an arity/3 handler call:

```eex
<button drab='click:button_clicked("<%= some_value =>")'>
```

and

```eex
<button drab="click:button_clicked('<%= some_value =>')">
```
both works in 0.7.6.

In 0.7.7 you can only use this form:

```eex
<button drab="click:button_clicked('<%= some_value =>')">
```

Is this intended?

**Comment by grych** [comment]
Not intended, both forms should work. Thanks for reporting!

**Comment by grych** [comment]
OK got it, thanks a lot, this is quite an unusual issue cause by how Floki renders back the html. I am raising an issue there, in the meantime please use the working form.

**Comment by OvermindDL1** [comment]
@grych Have you looked at [ModestEx](https://github.com/f34nk/modest_ex), it wraps the modest library, which is designed for HTML manipulation and as such it might work in a more expected way?

**Comment by grych** [comment]
Hey @OvermindDL1, I even spoke to F34nk during the ConfEU :) Yes, I am planning to give it a try.

**Comment by grych** [comment]
Fixed in floki 0.20.2; please upgrade to this version, next release of Drab will require >=0.20.2

**Comment by guidotripaldi** [comment]
checked. Works great!
